amrplot.py

```diff
@@ -361,11 +366,8 @@ def set_cmd(ss, pp):
             ss.center = (x, y, z)
 
     elif setting == "normal":
-        if len(pp) == 2:
-            if pp[1] not in ["x", "y", "z"]:
-                print("invalid normal vector direction")
-            else:
-                ss.normal = pp[1]
+        if pp[1] in {'x', 'y', 'z'}:
```

Comment:
why a dictionary here?

Reply:
It's actually a set, which is technically faster than a list when using the 'in' operator. For three elements it doesn't really matter though - I forgot I changed that.

Reply:
oh neat